linz.js

```diff
@@ -468,9 +474,15 @@ Linz.prototype.bootstrapExpress = function (cb) {
 		this.app.use(expressSession({ secret: this.get('cookie secret') }));
 	}
 
-	// initialize passport
-	this.app.use(this.get('admin path'), passport.initialize());
-	this.app.use(this.get('admin path'), passport.session());
+	// setup connect-flash
+	this.app.use(flash());
+
+	// configure passport
+	libPassport(this.passport);
```

Comment:
Yeah, that is a good point. I think libPassport needs to be updated so that it doesn't create a `serializeUser` or `deserializeUser` if they already exist.